src/Chapter1.hs

```diff
@@ -209,31 +209,31 @@ So, the output in this example means that 'False' has type 'Bool'.
 > Try to guess first and then compare your expectations with GHCi output
 
 >>> :t True
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+True :: Bool
 >>> :t 'a'
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+'a' :: Char
 >>> :t 42
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+42 :: Num t => t
```

Comment:
Unfortunately, the type variable is different on different GHC versions. Yeah, I know, non-deterministic algorithms... To make doctests pass, you need to change the type variable to a different letter :exploding_head: 
```suggestion
42 :: Num p => p
```